core/cas-server-core-util/src/main/java/org/apereo/cas/util/gen/AbstractRandomStringGenerator.java

```diff
@@ -0,0 +1,74 @@
+package org.apereo.cas.util.gen;
+
+import java.security.SecureRandom;
+
+/**
+ * Implementation of the RandomStringGenerator that allows you to define the
```

Comment:
Turn this into a `@link` javadoc tag please. Lest the file gets moved/renamed, we should be able to catch it.